.github/actions/automatic-releases/action.yml

```diff
@@ -0,0 +1,20 @@
+# Definition of the github action
+# as per https://docs.github.com/en/actions/creating-actions/creating-a-docker-container-action
+
+name: 'laminas/automatic-releases'
+description: 'Automates automatic releases for semver-compliant repositories'
+
+inputs:
+  command-name:
+    description: |
+      Command to execute: one of
+       * `laminas:automatic-releases:release`
+       * `laminas:automatic-releases:create-merge-up-pull-request`
+       * `laminas:automatic-releases:switch-default-branch-to-next-minor`
+    required: true
+
+runs:
+  using: 'docker'
+  image: '../../../Dockerfile'
+  args:
+    - ${{ inputs.command-name }}
```

Comment:
What I discovered is that while the image is in a a directory relative to this one, the actual build happens relative to `$GITHUB_WORKSPACE`, so everything works correctly. By doing this, we get the benefits of whatever changes we've made for this release... when creating the release.